data/frame_json.go

```diff
@@ -31,7 +31,7 @@ const jsonKeySchema = "schema"
 const jsonKeyData = "data"
 
 func init() { //nolint:gochecknoinits
-	jsoniter.RegisterTypeEncoder("data.Frame", &dataFrameCodec{})
+	sdkjsoniter.RegisterTypeEncoder("data.Frame", &dataFrameCodec{})
 	jsoniter.RegisterTypeDecoder("data.Frame", &dataFrameCodec{})
```

Comment:
I cannot import this from `sdkjsoniter`. I am getting type errors. I couldn't find a workaround. Suggestions are welcome.

Reply:
I think you may had a typo, this works fine:

```
--- a/data/utils/jsoniter/jsoniter.go
+++ b/data/utils/jsoniter/jsoniter.go
@@ -28,6 +28,7 @@ var (
 
 type Stream = j.Stream
 type ValEncoder = j.ValEncoder
+type ValDecoder = j.ValDecoder
 
 type Iterator struct {
        // named property instead of embedded so there is no
@@ -178,3 +179,7 @@ func ParseString(cfg j.API, input string) (*Iterator, error) {
 func RegisterTypeEncoder(typ string, encoder ValEncoder) {
        j.RegisterTypeEncoder(typ, encoder)
 }
+
+func RegisterTypeDecoder(typ string, decoder ValDecoder) {
+       j.RegisterTypeDecoder(typ, decoder)
+}
```

Reply:
Wow. Thank you. I think, as you said, I had a typo somewhere.

Reply:
https://github.com/grafana/grafana-plugin-sdk-go/pull/841/commits/c38be49172e0782715fa67c743fc046f96066f70